src/main/frontend/format/block.cljs

```diff
@@ -385,6 +385,39 @@
                               [:block/name (string/lower-case tag)]) tags))
     block))
 
+(defn- remove-indentation-spaces
+  [s level]
+  (let [level (inc level)
+        lines (string/split-lines s)
+        [f & r] lines
+        body (map (fn [line]
+                    (if (string/blank? (util/safe-subs line 0 level))
+                      (util/safe-subs line level)
+                      line))
+                  r)
+        content (cons f body)]
+    (string/join "\n" content)))
+
+(defn- get-block-content
+  [utf8-content block format]
+  (def aaa [utf8-content block format])
+  (let [meta (:meta block)
+        content (if-let [end-pos (:end-pos meta)]
+                  (utf8/substring utf8-content
+                                  (:start-pos meta)
+                                  end-pos)
+                  (utf8/substring utf8-content
+                                  (:start-pos meta)))]
+    (let [content (when content
+                    (let [content (text/remove-level-spaces content format)]
+                      (if (or (:pre-block? block)
+                              (= (:format block) :org))
+                        content
+                        (remove-indentation-spaces content (:level block)))))]
+      (if (= format :org)
+        content
+        (text/->new-properties content)))))
```

Comment:
I think this should be `property/->new-properties` now :)

Reply:
yes, fixed